modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/TypeScriptAngularClientCodegen.java

```diff
@@ -200,6 +205,10 @@ public void processOpts() {
             taggedUnions = Boolean.parseBoolean(additionalProperties.get(TAGGED_UNIONS).toString());
         }
 
+        if (additionalProperties.containsKey(SUBTYPE_VISITOR)) {
+            subtypeVisitor = Boolean.parseBoolean(additionalProperties.get(SUBTYPE_VISITOR).toString());
+        }
+
```

Comment:
As you commented above, this change is incompatible with `taggedUnions` - could we add a check for this here, and output an error & terminate if both options are set?

I am unsure how to implement this though - maybe @macjohnny or @wing328 know more and could shed some light on how this is usually done in other generators.

Reply:
The current implementation ignores the subtypeVisitor property when taggedUnions is set, it doesn't (or at least shouldn't) generate non-compilable code.

This could of course be changed in order to throw an exception if that is preferred.

Reply:
I think an exception or at least a warning would be useful for users

Reply:
I have added support for generating a visitor for tagged unions, so there is no need for this warning anymore.